_button.scss

```diff
@@ -0,0 +1,71 @@
+// button
+// ===
+
+// Configurable Variables
+// ---
+
+//Button Styles
+//---
+
+$button__padding: $vertical-input-padding $horizontal-input-padding !default;
+
+$button__border: if(variable-exists(border), $border, 1px solid $neutral-50) !default;
+$button__border-radius: if(variable-exists(border-radius), $border-radius, 0) !default;
+
+
+$button__font-color: inherit !default;
+$button__font-family: inherit !default;
+$button__font-size: inherit !default;
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'level/level';
+
+
+// Button root
+// ---
+
+.c-button {
+    min-height: $tap-size;
+    display:inline-block;
+    margin: 0;
+    cursor:pointer;
+    background: none;
+
+    text-align:center;
+    border:none;
+    line-height: normal;
+    white-space: normal;
+
+    span {
+        font-size: $button__font-size;
+        font-family: $button__font-family;
+
+        border: $button__border;
+        border-radius: $button__border-radius;
+        padding: $button__padding;
+
+        display:inline-block;
+    }
+
+    user-select:none;
+    -webkit-appearance: none;
```

Comment:
There is nothing special about the design of this component, except for one thing: the ability for the tap target to be bigger than the visible size of the button. The challenge with this is that ideally the tap size would not affect layout, allowing things to be a little more compact without impacting usability.

The styles here get us halfway there: they allow the button to *potentially* be taller than the visible portion, but the tappable portion controls layout, rather than the visible portion.

I wasn’t sure how to do this myself, either, so [I made a test page](http://codepen.io/ry5n/pen/WvLJrg?editors=110). My first idea was to use generated content, which seems to work just fine (verified in Chrome/Android, iOS Safari 7+). To ensure consistency of the visual height across components, we’ll want to add a variable for this to the stencil-variables repo.